addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1089,9 +1117,12 @@ const SelectUserValueWidget = BaseSelectionUserValueWidget.extend({
         }
 
         if (!this.menuTogglerEl.classList.contains('active')) {
-            this.open();
+            this.open().then(() => {
+                this._adjustDropdownPosition();
+            });
```

Comment:
If open is async and its return promise is not awaited somewhere, we are probably doing something wrong (maybe not but probably).

Reply:
I don't think there is a problem.. all `open()` are now async.. maybe I didn't understand something?

Reply:
https://github.com/odoo/odoo/pull/135788/files#r1345528284